Lib/pathlib.py

```diff
@@ -1440,6 +1440,14 @@ def expanduser(self):
 
         return self
 
+    @classmethod
+    def relative(cls, path, depth=1):
```

Comment:
1. This name is too similar to `relative_to`.
2. The name doesn't explain it's relative to the caller.
3. It's not relative to the "caller file" but to the "caller directory".
4. The result isn't actually a relative path.

Reply:
Really cannot find a better name. It is `rel_path` in the internal package. 
I would really like to make it as short as possible, like `rel` but it isn't more descriptive at all.